includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -864,6 +867,24 @@ public function needs_setup() {
 		return parent::needs_setup() || ! empty( $account_status['error'] ) || ! $account_status['paymentsEnabled'];
 	}
 
+	/**
+	 * Get the agentic commerce payment provider identifier.
+	 *
+	 * @return string Payment provider identifier.
+	 */
+	public function get_agentic_commerce_provider() {
+		return 'stripe';
```

Comment:
Since we need to figure out auth in WooCommerce core, this should also probably live in core as well alongside the auth bit.

Reply:
Disregard. I understand how this fits together now.